spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1348,57 +1372,92 @@
+                    )
                   end
                 end
-              end
 
-              it 'deactivates and sets fraud related fields of an expired enrollment' do
-                stub_request_expired_id_ipp_proofing_results
+                it 'deactivates the associated profile' do
+                  expect(pending_enrollment.profile.in_person_verification_pending_at).not_to be_nil
 
-                job.perform(Time.zone.now)
+                  job.perform(Time.zone.now)
+                  pending_enrollment.reload
+                  expect(pending_enrollment.profile.in_person_verification_pending_at).to be_nil
+                  expect(pending_enrollment.profile.active).to be false
+                  expect(pending_enrollment.profile.deactivation_reason).
+                    to eq('verification_cancelled')
+                end
 
-                profile = pending_enrollment.reload.profile
-                expect(profile).not_to be_active
-                expect(profile.fraud_review_pending_at).to be_nil
-                expect(profile.fraud_rejection_at).not_to be_nil
-                expect(job_analytics).to have_logged_event(
-                  :idv_ipp_deactivated_for_never_visiting_post_office,
-                )
+                it 'deactivates and sets fraud related fields of an expired enrollment' do
+                  stub_request_expired_id_ipp_proofing_results
+
+                  job.perform(Time.zone.now)
+
+                  profile = pending_enrollment.reload.profile
+                  expect(profile).not_to be_active
+                  expect(profile.fraud_review_pending_at).to be_nil
+                  expect(profile.fraud_rejection_at).not_to be_nil
+                  expect(job_analytics).to have_logged_event(
+                    :idv_ipp_deactivated_for_never_visiting_post_office,
+                  )
+                end
               end
             end
           end
         end
 
         describe 'Proofed with secondary id' do
-          let(:pending_enrollment) do
-            create(
-              :in_person_enrollment, :pending
-            )
+          let!(:pending_enrollments) do
+            ['BALTIMORE', 'FRIENDSHIP', 'WASHINGTON', 'ARLINGTON', 'DEANWOOD'].map do |name|
+              create(
+                :in_person_enrollment,
+                :pending,
+                :with_notification_phone_configuration,
+                issuer: 'http://localhost:3000',
+                selected_location_details: { name: name },
+                sponsor_id: usps_ipp_sponsor_id,
+              )
+            end
+          end
+          let(:pending_enrollment) { pending_enrollments.first }
+
+          before do
+            enrollment_records = InPersonEnrollment.where(id: pending_enrollments.map(&:id))
+            # Below sets in_person_verification_pending_at
+            # on the profile associated with each pending enrollment
+            enrollment_records.each do |enrollment|
```

Comment:
Is it possible to update the `:in_person_enrollment` `:pending` factory trait and set the profile `in_person_verification_pending_at` timestamp in there, maybe similar to what Andrew did [here](https://github.com/18F/identity-idp/pull/11129/files#diff-97e661aa6e51c41e8d02b50ee1749b96cd4042ad1eb8a88fca9c9f68e2ce1f42R219) in the users factory?

I know we have a ticket to clean up the `in_person_enrollments` factory and am okay with also doing the suggested work while working on that ticket.

Reply:
Thanks for adding that, @shanechesnutt-ft!